<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="fontLabel">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Current font :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeFontButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Change &amp;font</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroupBox">
     <property name="title">
      <string>Syntax highlight colors</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="3">
         <widget class="QPushButton" name="bColorButton">
          <property name="text">
           <string>B90. C30.</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="zColorButton">
          <property name="text">
           <string>Z26.45</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="dhtColorButton">
          <property name="text">
           <string>D3 H7 T1</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="fsColorButton">
          <property name="text">
           <string>F100 S500</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="operatorColorButton">
          <property name="text">
           <string>+-=/*</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="keyWordColorButton">
          <property name="text">
           <string>SPFA</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="lColorButton">
          <property name="text">
           <string>L100</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="rColorButton">
          <property name="text">
           <string>R10</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="nColorButton">
          <property name="text">
           <string>N1 N2 N3</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="gColorButton">
          <property name="text">
           <string>G00 G90</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="commentColorButton">
          <property name="text">
           <string>(Comment)</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="progNameColorButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>%1000 O1234</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="mColorButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>M3 M8</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="aColorButton">
          <property name="text">
           <string>A120.</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="macroColorButton">
          <property name="text">
           <string>SIN() ABS()</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="capsLockCheckBox">
          <property name="text">
           <string>&amp;Internal CAPS LOCK</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="syntaxHCheckBox">
          <property name="text">
           <string>&amp;Syntax highlight</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="curLineColorButton">
          <property name="toolTip">
           <string>Highlighed current line color</string>
          </property>
          <property name="text">
           <string>Hi&amp;ghlighed current line color</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="tabbedModecheckBox">
          <property name="text">
           <string>&amp;Editor tabbed mode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="underlineCheckBox">
          <property name="text">
           <string>Underline changes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="underlineColorButton">
          <property name="toolTip">
           <string>Underline color</string>
          </property>
          <property name="text">
           <string>U&amp;nderline color</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>&amp;Default</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>changeFontButton</tabstop>
  <tabstop>rColorButton</tabstop>
  <tabstop>lColorButton</tabstop>
  <tabstop>operatorColorButton</tabstop>
  <tabstop>mColorButton</tabstop>
  <tabstop>progNameColorButton</tabstop>
  <tabstop>keyWordColorButton</tabstop>
  <tabstop>aColorButton</tabstop>
  <tabstop>dhtColorButton</tabstop>
  <tabstop>bColorButton</tabstop>
  <tabstop>macroColorButton</tabstop>
  <tabstop>gColorButton</tabstop>
  <tabstop>nColorButton</tabstop>
  <tabstop>fsColorButton</tabstop>
  <tabstop>zColorButton</tabstop>
  <tabstop>commentColorButton</tabstop>
  <tabstop>capsLockCheckBox</tabstop>
  <tabstop>curLineColorButton</tabstop>
  <tabstop>syntaxHCheckBox</tabstop>
  <tabstop>tabbedModecheckBox</tabstop>
  <tabstop>underlineCheckBox</tabstop>
  <tabstop>underlineColorButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>defaultButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
